packages/chrome.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Install a custom Chrome version, https://www.google.com/chrome
+# https://googlechromelabs.github.io/chrome-for-testing/
+# Removes the pre-installed stable version of Chrome and replaces it with the specified version
+# 
+# Add at least the following environment variables to your project configuration
+# (otherwise the defaults below will be used).
+# * CHROME_VERSION
+# 
+# Include in your builds via
+# \curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/chrome-beta.sh | bash -s
```

Comment:
This link would work only after this PR gets merged. This can be tested using the script here: https://raw.githubusercontent.com/trainer-rx/codeship-scripts/script-for-custom-chrome-version/packages/chrome.sh